test/recipes/controls/aws_parallelcluster_slurm/config_slurmd_systemd_service_spec.rb

```diff
@@ -0,0 +1,38 @@
+# Copyright:: 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+
+control 'systemd_slurmd_service' do
+  title 'Check the basic configuration of the systemd slurmd service'
+
+  describe 'Check that slurmd service is defined'
+  describe service('slurmd') do
+    it { should be_installed }
+  end
+
+  describe 'Check slurmd systemd "after" dependencies'
+  describe command('systemctl list-dependencies --after --plain slurmd.service') do
+    its('stdout') { should include "munge.service" }
+    its('stdout') { should include "munge.service" }
```

Comment:
Duplicated line.
Do you expected another service to be present in the list of services ?

Reply:
I will remove it in the next commit.